src/CarbonAware.WebApi/src/Configuration/ServiceCollectionExtensions.cs

```diff
@@ -7,13 +9,23 @@ public static void AddMonitoringAndTelemetry(this IServiceCollection services, I
         
         var envVars = configuration?.GetSection(CarbonAwareVariablesConfiguration.Key).Get<CarbonAwareVariablesConfiguration>();
         var telemetryProvider = GetTelemetryProviderFromValue(envVars?.TelemetryProvider);
-
+        var logger = CreateConsoleLogger(configuration);
         switch (telemetryProvider) {
             case TelemetryProviderType.ApplicationInsights:
             {
-                if (!String.IsNullOrEmpty(configuration?["ApplicationInsights_Connection_String"]))
+                if (!String.IsNullOrEmpty(configuration?["ApplicationInsights_Connection_String"])) 
                 {
+                    logger.LogInformation("Application Insights connection string found");
                     services.AddApplicationInsightsTelemetry();
+                } 
+                else if (!String.IsNullOrEmpty(configuration?["ApplicationInsights_Instrumentation_Key"])) 
```

Comment:
This constant is wrong. It should be: `AppInsights_InstrumentationKey`

Reply:
Verified this again just now with 'ApplicationInsights_Instrumentation_Key' and I see the logs flowing in to AppInsights

Reply:
It works because it is passed as an argument of the method and that method signature, the string is the instrumentation key. The comment is more towards standard of using env vars following the way how that variable is consumed with other Azure Services.

Reply:
So, that means the connection string env variable should also be changed to `AppInsights_ConnectionString`?

Reply:
yeah, that was so confusing to see the beginning of the env. vars not match!

https://docs.microsoft.com/en-us/azure/azure-functions/functions-app-settings